README.md

```diff
@@ -325,6 +326,37 @@ Workflow issues may be submitted to the `ITKRemoteModuleBuildTestPackageAction`
 [issue tracker](https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/issues)
 on GitHub.
 
+## Frequently Asked Questions
+
+### What target platforms and architectures are supported?
+
+The `build-test-cxx` workflow supports building and running C++ tests on GitHub Actions runner images for
+Windows, Linux, and macOS. Image details are available [here](https://github.com/actions/runner-images).
+
+The `build-test-package-python.yml` workflow supports Python package generation for the following platforms and architectures:
+- Windows 10 x86_64 platforms
+- Windows 11 x86_64 platforms
+- macOS 10.9+ x86_64 platforms
+- Linux glibc 2.17+ (E.g. Ubuntu 18.04+) x86_64 platforms
+- Linux glibc 2.28+ (E.g. Ubuntu 20.04+) aarch64 (ARMv8) platforms
+
+The workflow does not support macOS aarch64 (ARM) packages for ITK external modules at this time.
+See [#528](https://github.com/github/roadmap/issues/528).
```

Comment:
#528 -> GitHub Roadmap Issue 528

to distinguish from issues in our repository